docs/devTools/README.md

```diff
@@ -0,0 +1,62 @@
+# Dev tools
+
+Local development tools for deploying the [any.sender contracts](https://github.com/PISAresearch/contracts.any.sender) and hosting a mock any.sender instance. We also have a live ropsten API for when you're ready to take it to the next level :)
+
+## Deploying contracts locally
+
+To deploy the contracts import the `deploy` object from the `@any-sender/client` package.  The deploy object has an `contracts` function which accepts a provider and returns a promise of the contracts. Call this at some point in your tests before wrapping objects with `any.sender()`.
```

Comment:
has a 

is there an ideal time to call it? 
- before setting up tests? at the start of a test? what does it actually deploy and why do they care?

Reply:
Fixed typo

Updated to say:
"Call this at the start of your tests before wrapping objects with `any.sender()`."